### ISSUE TITLE
fraction methods are not typed

### ISSUE THREAD

**Issue by yukulele** [josdejong/mathjs#3317]
**Describe the bug**
`Fraction` type only exposes `s`, `n` add `d` properties

[mathjs/types/index.d.ts:4018](https://github.com/josdejong/mathjs/blob/develop/types/index.d.ts#L4018):
```ts
export interface Fraction {
  s: number
  n: number
  d: number
}
```
so we can't use methods

**To Reproduce**
```ts
math.fraction('1.5').toFraction() // "3/2"
```
```console
> tsc
error TS2339: Property 'toFraction' does not exist on type 'Fraction'.

9 math.fraction('1.5').toFraction()
                       ~~~~~~~~~~
```

**Comment by josdejong** [comment]
Thanks for reporting, we should improve the TypeScript definitions. Ideally, we can import and use type definitions from Fraction.js.

Who can help fix the TypeScript definitions of fractions?

**Comment by yukulele** [comment]
I think you might need to update the [fraction.js version](https://github.com/rawify/Fraction.js/tags) first, which is out [outdated](https://github.com/josdejong/mathjs/blob/develop/package.json#L32).

**Comment by josdejong** [comment]
Yes, that is addressed via https://github.com/josdejong/mathjs/pull/3283, it is a breaking change planned for `v14`.

**Comment by josdejong** [comment]
Fixed now in `v14.0.1`.
